Ports/p7zip/package.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port="p7zip"
+version="17.04"
+useconfigure=true
+auth_type=sha256
+files="https://github.com/jinfeihan57/p7zip/archive/refs/tags/v${version}.tar.gz p7zip-${version}.tar.gz ea029a2e21d2d6ad0a156f6679bd66836204aa78148a4c5e498fe682e77127ef"
+configopts="-DCMAKE_TOOLCHAIN_FILE=$SERENITY_SOURCE_DIR/Toolchain/CMake/CMakeToolchain.txt"
+
+post_fetch() {
+    mv ${port}-${version}/CPP/7zip/CMAKE/7za/CMakeLists.txt ${port}-${version}/CPP/7zip/CMAKE/7za/CMakeLists_CR.txt
+    sed -e 's/\r//' ${port}-${version}/CPP/7zip/CMAKE/7za/CMakeLists_CR.txt > ${port}-${version}/CPP/7zip/CMAKE/7za/CMakeLists.txt
+    rm ${port}-${version}/CPP/7zip/CMAKE/7za/CMakeLists_CR.txt
+}
+
+configure() {
+    mkdir -p build && cd build
+    cmake ../${port}-${version}/CPP/7zip/CMAKE $configopts
+}
+
+build() {
+    make $makeopts
+}
+
+install() {
+    cp -r "bin/Codecs" "${SERENITY_INSTALL_ROOT}/usr/local/bin"
+    cp "bin/7z_" "${SERENITY_INSTALL_ROOT}/usr/local/bin"
+    cp "bin/7z.so" "${SERENITY_INSTALL_ROOT}/usr/local/bin"
+    cp "bin/7za" "${SERENITY_INSTALL_ROOT}/usr/local/bin"
+    cp "bin/7zCon.sfx" "${SERENITY_INSTALL_ROOT}/usr/local/bin"
+    cp "bin/7zr" "${SERENITY_INSTALL_ROOT}/usr/local/bin"
+}
```

Comment:
These seem to rely on `configure` `cd`'ing? Please don't do that as these steps can also be run in isolation. Instead, set the [`workdir`](https://github.com/SerenityOS/serenity/blob/master/Ports/README.md#workdir) variable and use [`run`](https://github.com/SerenityOS/serenity/blob/master/Ports/README.md#run) for the individual commands.